### ISSUE TITLE
HiddenSelect aria-hidden elements are not focusable nor contain focusable elements

### ISSUE THREAD

**Issue by JamesParkinSonos** [adobe/react-spectrum#4665]
# 🐛 Bug Report 

The Deque Axe Tool is returning this serious issue when using a HiddenSelect component. 

`ARIA hidden element must not be focusable or contain focusable elements`

## 🤔 Expected Behavior

No issue reported

## 😯 Current Behavior

This issue can be replicated on the react-aria/useSelect page (https://react-spectrum.adobe.com/react-aria/useSelect.html) with the axe DevTools installed. After scanning the entire page 7 serious issues are raised.

An example message is:

Ensures aria-hidden elements are not focusable nor contain focusable elements

`[more informationLink opens in a new window](https://dequeuniversity.com/rules/axe/4.7/aria-hidden-focus?application=AxeChrome)`
Element Location:
`#example-0 > .example_d569a7.spectrum_2a241c.spectrum_b37d53 > div > div[aria-hidden="true"]`
`<div aria-hidden="true" style="border: 0px; clip: rect(0px, 0px, 0px, 0px); clip-path: inset(50%); height: 1px; margin: -1px; overflow: hidden; padding: 0px; position: absolute; width: 1px; white-space: nowrap;">`
To solve this problem, you need to fix the following:
Focusable content should be disabled or be removed from the DOM

Related Node
`<input type="text" tabindex="0" style="font-size: 16px;">`

## 💁 Possible Solution

The input tabindex should be -1 
`
<input type="text" tabindex="-1" style="font-size: 16px;">
`

## 🔦 Context

We are using the HiddenSelect component and want to be as accessible as possible.

## 🌍 Your Environment

| Software         | Version(s) |
| ---------------- | ---------- |
| react-aria   | 3.25.0
| Browser          | Chrome 114.0.5735.106
| Operating System | MacOS 13.4

**Comment by LFDanLu** [comment]
We are tracking this false positive in our wiki [here](https://github.com/adobe/react-spectrum/wiki/Known-accessibility-false-positives#picker). aXe doesn't know that we manually handle focus in such a way that the HiddenSelect will move focus to the actual Select element so it throws that error.

**Comment by JamesParkinSonos** [comment]
Thank you for the instant reply. So, there shouldn't be an impact on accessibility?
